### DISCUSSION TITLE
Remove support for some languages

### DISCUSSION THREAD

**Discussion by marrus-sh** [cat-KIBI-glitch/mastodon#23] [Request for Comments]
Kibicat Mastodon has a much smaller intended audience than upstream Mastodon. We may want to limit supported languages (for the user interface) to only those which we have the resources to maintain at a high level of quality, rather than depend on upstream localization files (which will not have full coverage of Kibicat features, and may not be, like, *good*).

There are pros and cons on both sides of this, but I think it's something we should discuss.

**Comment by aescling** [comment]
i think our priorities when it comes to this problem will depend on how many people are using our software and we should be prepared to change it in the future if project goals or reach changes

i think insofar as we are working on it for GlitchCat though, then we need only support the language(s) our users do, so long as we don’t like, break potential internationalization in the future (and ofc develop new features with support for internationalization in mind)

**Reply by marrus-sh** [reply]
It’s *way* easier to add a locale than remove one, so.
